packages/discord.js/typings/index.d.ts

```diff
@@ -3738,6 +3738,7 @@ export type CommandOptionChannelResolvableType = ApplicationCommandOptionType.Ch
 
 export type CommandOptionChoiceResolvableType =
   | ApplicationCommandOptionType.String
+  | ApplicationCommandOptionType.Boolean
```

Comment:
You can't add Choice to the Boolean option. It shouldn't be there.

Reply:
I see now. Choices for Booleans don't exist. Should have read <https://discordjs.guide/interactions/slash-commands.html#choices>.

By that logic, it's probably also completely wrong to inherit from `ApplicationCommandChoicesData` and `ApplicationCommandChoicesOption` respectively.

Are there any respective "neutral" types to inherit from? Is `BaseApplicationCommandOptionsData` what I need here?

Reply:
Yes, it is.